contributing/DEVELOPING.md

```diff
@@ -89,7 +89,8 @@ The Quick Start Guide's  [One-time setup](getting-started-quick.md#one-time-setu
 | `node build-system/pr-check.js`                                         | Runs all tests that will be run upon pushing a CL.                     |
 | `gulp ava`                                                              | Run node tests for tasks and offline/node code using [ava](https://github.com/avajs/ava). |
 | `gulp todos:find-closed`                                                | Find `TODO`s in code for issues that have been closed. |
-| `gulp visual-diff`                                                      | Runs all visual diff tests on local Chrome. Requires `gulp build` to have been run. Also requires `PERCY_PROJECT` and `PERCY_TOKEN` to be set as environment variables. |
+| `gulp visual-diff`                                                      | Runs all visual diff tests on local Chrome. Requires `PERCY_PROJECT` and `PERCY_TOKEN` to be set as environment variables. |
+| `gulp visual-diff --nobuild`                                            | Same as above, but without re-build. |
```

Comment:
We don't list `gulp visual-diff --verify_status` here, but that's probably okay since it's used on Travis. Your call on whether it's worth adding so it can be done during local dev.

Reply:
I've noticed that too. I don't think it's an issue, `--verify_status` is internal, most developers, at the moment, do not have access to our Percy project so the knowledge of whether there are visual differences won't help them much. If we ever get public read-only access to Percy we can add that line here